src/modal-manager/ModalManager.d.ts

```diff
@@ -0,0 +1,17 @@
+import { CSSProperties, PureComponent } from 'react';
+import { Type } from 'tcomb';
+
+export type ModalManagerProps = {
+  children?: any,
+  transitionStyles?: { [key: string]: CSSProperties },
```

Comment:
this could be better typed as `TransitionWrapperProps .TransitionStyles` from `TransitionWrapper.d.ts`